shared/reducers/fs.js

```diff
@@ -86,12 +86,16 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
       return state.mergeIn(['flags'], action.payload)
     case FsGen.installKBFS:
       return state.mergeIn(['flags'], {kbfsInstalling: true})
+    case FsGen.syncingStatus:
+      return state.mergeIn(['flags'], {syncing: action.payload.isSyncing})
```

Comment:
I like to name the action flags the same thing as the state flags, allowing `state.mergeIn(['flags'], action.payload)`